includes/class-post-types.php

```diff
@@ -572,6 +598,48 @@ public static function register_ap_actor_rest_field() {
 		);
 	}
 
+	/**
+	 * Register custom query parameters for the ap_actor REST endpoint.
+	 *
+	 * @param array $params The existing collection parameters.
+	 *
+	 * @return array Modified collection parameters.
+	 */
+	public static function register_ap_actor_rest_query_params( $params ) {
+		$params['activitypub_following'] = array(
+			'description'       => \__( 'Filter actors by who they follow (WordPress user ID).', 'activitypub' ),
+			'type'              => 'integer',
+			'sanitize_callback' => 'absint',
+			'validate_callback' => 'is_user_member_of_blog',
+		);
+
+		return $params;
+	}
+
+	/**
+	 * Filter the ap_actor REST query to support meta filtering.
+	 *
+	 * @param array            $args    Query arguments.
+	 * @param \WP_REST_Request $request The REST request.
+	 *
+	 * @return array Modified query arguments.
+	 */
+	public static function filter_ap_actor_rest_query( $args, $request ) {
+		$following = $request->get_param( 'activitypub_following' );
+
+		// Only filter if user ID provided and social graph is visible or current user can edit that user.
+		if ( $following && ( Actors::show_social_graph( $following ) || \current_user_can( 'edit_user', $following ) ) ) {
```

Comment:
This check `\current_user_can( 'edit_user', $following )` might also not work for the Blog-User (id `0`).

Reply:
Fixed in [5660fc9](https://github.com/Automattic/wordpress-activitypub/pull/2625/commits/5660fc9689fae6c673bc071692963ad31fffdf78).